src/repo/fs/blocks.rs

```diff
@@ -100,6 +100,57 @@ impl FsBlockStore {
             }
         }
     }
+
+    /// Implementation moved here from inside the `impl BlockStore for FsBlockStore` which is
```

Comment:
Is this move still required? I expected the `Cargo.lock` removal to be the only change.

Reply:
> Weird thing.

Yeah, I think there's a bug to be reported to somewhere in here, but I would need to double check the async-trait 0.1.43 PR and all other PRs referring to it, maybe ask around in the chats...